cmd/macoffice/generate.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/vulnerabilities/macoffice"
+)
+
+func panicif(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+// Generates Mac Office release notes metadata in JSON format, to be used by our vulnerability process.
+func main() {
```

Comment:
After we merge this, there is a PR coming with this on https://github.com/fleetdm/nvd, right?

Reply:
Correct